jwst/cube_build/cube_build_io_util.py

```diff
@@ -99,7 +99,7 @@ def read_cubepars(par_filename,
                                                     table_sroi, table_wroi,
                                                     table_scalerad)
 
-            #  read in wavelength table for modified shepard method 1/r weighting
+            #  read in wavelength table for multi-banddata
```

Comment:
Is "multi-banddata" supposed to be all one word, or should there be a white space between "band" and "data"?